client/src/panels/panel_declarations/services/provided_services_list.js

```diff
@@ -6,25 +6,80 @@ import {
   InfographicPanel,
   declare_panel,
 } from "../shared.js";
+import { FancyUL } from "../../../components";
 
 const { text_maker, TM } = create_text_maker_component(text);
 
-const ProvidedServicesListPanel = ({ panel_args }) => (
-  <div>
-    <TM
-      k="list_of_provided_services_desc"
-      args={{
-        subject_name: panel_args.subject.name,
-        num_of_services: panel_args.services.length,
-      }}
-    />
-    {_.map(panel_args.services, ({ name, id, org_id }) => (
-      <a key={id} href={`#dept/${org_id}/service-panels/${id}`}>
-        {name}
-      </a>
-    ))}
-  </div>
-);
+class ProvidedServicesListPanel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { query: "" };
+  }
+
+  render() {
+    const { panel_args } = this.props;
+
+    const filtered_services = _.filter(panel_args.services, (service) => {
+      if (
+        _.includes(service.name.toLowerCase(), this.state.query.toLowerCase())
+      ) {
+        return true;
+      } else if (
+        _.includes(
+          service.service_type.toLowerCase(),
+          this.state.query.toLowerCase()
+        )
+      ) {
+        return true;
+      }
+      return false;
+    });
+
+    return (
+      <div>
+        <TM
+          k="list_of_provided_services_desc"
+          args={{
+            subject_name: panel_args.subject.name,
+            num_of_services: panel_args.services.length,
+          }}
+        />
+        <input
+          aria-label={text_maker("explorer_search_is_optional")}
+          className="form-control input-lg"
+          type="text"
+          style={{ width: "100%", marginBottom: "1rem" }}
+          placeholder={text_maker("filter_results")}
+          onChange={(evt) => this.setState({ query: evt.target.value })}
+          value={this.state.query}
+        />
+        <FancyUL>
+          {_.map(
+            filtered_services,
+            ({ name, id, org_id, service_type, description }) => (
+              <React.Fragment key={id}>
+                <a href={`#dept/${org_id}/service-panels/${id}`}>{name}</a>
+                <p>{description}</p>
+                <div
+                  style={{
+                    display: "flex",
+                    fontSize: "14px",
+                    justifyContent: "space-between",
+                  }}
+                >
+                  <div className="tag-badge">{service_type}</div>
+                  <a href={`#dept/${org_id}/service-panels/${id}`}>
```

Comment:
You should try styling this and see how it looks, I was originally thinking of styling it with something like `btn-ib-primary`